Makefile

```diff
@@ -35,30 +70,30 @@ release: \
 # matchbox
 
 bin/linux-amd64/matchbox:
```

Comment:
Both the binary name and architecture are used in the recipe. It could probably be parsed out, but for now this release process is unchanged and fairly readable.